test/entropic/sinkhorn_stabilized.jl

```diff
@@ -0,0 +1,100 @@
+using OptimalTransport
+
+using Distances
+using ForwardDiff
+using LogExpFunctions
+using PythonOT: PythonOT
+
+using LinearAlgebra
+using Random
+using Test
+
+const POT = PythonOT
+
+Random.seed!(100)
+
+@testset "sinkhorn_stabilized.jl" begin
+    # size of source and target
+    M = 250
+    N = 200
+
+    # create two random histograms
+    μ = normalize!(rand(M), 1)
+    ν = normalize!(rand(N), 1)
+
+    # create random cost matrix
+    C = pairwise(SqEuclidean(), rand(1, M), rand(1, N); dims=2)
```

Comment:
Just curious. Any reason why doing `C = pairwise(SqEuclidean(), rand(1, M), rand(1, N); dims=2)` instead of `C = pairwise(SqEuclidean(), rand(M), rand(N); dims=1)`?